test/parallel/test-http-pipe-fs.js

```diff
@@ -39,7 +43,7 @@ const server = http.createServer(common.mustCall(function(req, res) {
 }, 2)).listen(0, function() {
   http.globalAgent.maxSockets = 1;
 
-  for (let i = 0; i < 2; ++i) {
+  for (let i = 0; i < NUMBER_OF_STREAMS; ++i) {
     (function(i) {
```

Comment:
I *think* this entire closure might be unnecessary now, because the countdown removes the only use of `i` that was not for debugging?

Reply:
@addaleax I just push a new commit that removes the unnecessary IIFE.